activitysim/abm/models/initialize_from_usim.py

```diff
@@ -660,10 +623,17 @@ def COLLPTE(colleges, zones):
 
 
 @orca.column('zones')
-def area_type():
-    # Integer, 0=regional core, 1=central business district, 2=urban business,
-    # 3=urban, 4=suburban, 5=rural
-    return 0  # Assuming all regional core
+def area_type(mpo_taz, zones):
+    
+    mpo = mpo_taz.to_frame(columns = ['geometry','area_type','ACRES'])
+    h3_gpd =  zones.to_frame(columns = ['geometry', 'area'])
+    
+    join = gpd.sjoin(h3_gpd, mpo, how = 'left',op='intersects')
+    join.area_type.fillna(5, inplace = True) #Fill non-matched areas with 5 (Rural areas)
+    join = join.groupby(['TAZ', 'area_type'])['ACRES'].sum().reset_index()
+    join = join.sort_values(['TAZ', 'ACRES'], ascending = True)
+    s = join.groupby('TAZ')['area_type'].last()
+    return s 
```

Comment:
this looks great. just curious as to how you mapped the MPO's 5 area type categories to Activitysim's 6. Is that documented anywhere?

Reply:
mapping is in area_type_impute.py